examples/skr/aks/README.md

```diff
@@ -44,9 +44,9 @@ You can follow the installation instructions for Azure CLI [here](https://learn.
 
 ### 2. Obtain An Attestation Endpoint
 
-Below are the MAA endpoints for the four regions Confidential Containers in AKS is currently available in. 
+Below are the MAA endpoints for the four regions Confidential Containers in AKS is currently available in.
```

Comment:
nit: end of sentence preposition, also "in" to "on"

Below are the MAA endpoints for the four regions in which  Confidential Containers on AKS is currently available